paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundActivity.kt

```diff
@@ -101,6 +107,14 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity(), ExternalPay
 
             val playgroundState by viewModel.state.collectAsState()
 
+            val customerSheet = playgroundState?.asCustomerState()?.let { customerPlaygroundState ->
```

Comment:
We want this to be unconditionally created like the others. Is that not possible with customer sheet? If not this integration shape is probably wrong :/